### PR TITLE
Simple support of composed Bridged Devices. 

### PR DESCRIPTION
**Pull request by AntonGrey**
#### Matter doesn't have support of  composed Bridged Devices.
The problem is that sdk currently provides no way to access(fill or read) PartsList on endpoints other than EP0. 
Link to [Topology or logical grouping](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/data_model/bridge-architecture.adoc#21-topology-or-logical-grouping) chapter which describe exposure of composed Bridged Devices.

#### Change overview
* Add link to root endpoint for endpoints of a composed Bridged Devices.
* Encode PartsList for a root endpoint of a composed Bridged Devices.

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

File: src/app/util/af-types.h [hunk 1]
```diff
@@ -415,6 +415,11 @@ struct EmberAfDefinedEndpoint
      * endpoint
      */
     chip::DataVersion * dataVersions = nullptr;
+
+    /**
+     * Root endpoint id for composed device type.
+     */
+    chip::EndpointId composedEpId = chip::kInvalidEndpointId;
```
Comment (mrjerryjohns) [inline src/app/util/af-types.h:422]: I think the term `composedxx` isn't quite conveying the fact that this is not denoting the set of child endpoints from a given parent, but the reverse: this is the endpoint ID of the parent.

`parentEndpointId` or something?

File: src/app/util/attribute-storage.h [hunk 2]
```diff
@@ -270,7 +270,8 @@ CHIP_ERROR emberAfSetDeviceTypeList(chip::EndpointId endpoint, chip::Span<const
 //
 EmberAfStatus emberAfSetDynamicEndpoint(uint16_t index, chip::EndpointId id, const EmberAfEndpointType * ep,
```
Comment (mrjerryjohns) [inline src/app/util/attribute-storage.h:271]: Please update the comments to this function indicating the use of the new argument.

**Review by mrjerryjohns — COMMENTED** [review]
Shouldn't the existing bridge app be updated to reflect this usage?

File: src/app/util/af-types.h (hunk 1 above)
Comment (AntonGrey) [reply src/app/util/af-types.h]: `parentEndpointId ` is well.

File: src/app/util/attribute-storage.h (hunk 2 above)
Comment (AntonGrey) [reply src/app/util/attribute-storage.h:273]: Sure.

**Pull request merged**